core/templates/dev/head/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -69,6 +70,11 @@ oppia.factory('threadDataService', [
         }
       }
     });
+    var payload = {
+      exploration_id: _expId,
+      thread_id: threadId
+    };
+    $http.post(_FEEDBACK_THREAD_VIEW_EVENT_URL, payload);
```

Comment:
Also, you can bring the payload inline to this call:

    $http.post(_FEEDBACK_THREAD_VIEW_EVENT_URL, {
      exploration_id: _expId,
      thread_id: threadId
    });